### ISSUE TITLE
Custom scalars treats as undefined

### ISSUE THREAD

**Issue by roman0x58** [victorgarciaesgi/simple-graphql-to-typescript#1]
Hi Victor, it seems like the `--customScalars` option didn't work correctly. I've been running sgts like 
```bash 
sgts --endpoint http://localhost:8080/graphql --output src/javascripts/shared/model/generated.ts --suffix Model --customScalars [{ "ObjectIdType": "ObjectIdType" }]
```
Finally in the output file I have found that all fields with `ObjectIdType` treats as `undefined`.

**Comment by victorgarciaesgi** [comment]
Thanks for the report @roman0x58 ! Looking into it

**Comment by victorgarciaesgi** [comment]
@hassanij Can you take a look?

**Comment by j-hassani** [comment]
I'll look into it too this week-end, thanks for the report @roman0x58 !!

**Comment by hassanij** [comment]
#ft/1

**Comment by victorgarciaesgi** [comment]
Should be fixed on #2

**Comment by victorgarciaesgi** [comment]
@roman0x58 update to 0.651
